os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/wdt_api.c

```diff
@@ -48,6 +48,42 @@ IRQn_Type WdgIrqNum = WDG4_IRQ;
   * @{
   */
 
+
+
+void WDG_Init(WDG_TypeDef *WDG, WDG_InitTypeDef *WDG_InitStruct)
```

Comment:
The function is newly added. Could you let me know who calla this?

Reply:
Dear @sunghan-chang,
This function is not newly added, it was originally at ameba_wdg.c where amebasmart_wdg_settimeout in porting layer will call this function to reset watchdog timer with new timeout value.. 
Thank you